Makefile

```
@@ -6,6 +6,10 @@ DOCKER_TAG=rigetti/forest:$(COMMIT_HASH)
 .PHONY: all
 all: dist
 
+.phony: bench
```

I think casing makes a difference?

```suggestion
.PHONY: bench
```